internal/util/diff/diff_test.go

```diff
@@ -246,9 +340,14 @@ func TestCommand_Diff(t *testing.T) {
 				DiffToolOpts: tc.diffOpts,
 				Output:       diffOutput,
 			}).Run(fake.CtxWithNilPrinter())
-			assert.NoError(t, err)
+			if !assert.NoError(t, err) {
+				t.FailNow()
+			}
 
 			filteredOutput := filterDiffMetadata(diffOutput)
+			if tc.hasLocalSubpackageChanges {
```

Comment:
Do we need to check this separately from the other output?

Reply:
There's certainly no other way to reconcile the dynamically generated local subpackage location with the diff output.

Reply:
Ah, the printed path will be an issue.